features/error/error_page.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Error Page
+
+  Scenario: For a 500 Error Page
+    Given a configuration of:
+    """
+      ActiveAdmin.register_page "Dashboard" do
+        content do
+          five_hundred_error
+        end
+      end
+      """
```

Comment:
Please bring all into one line. See for example: https://github.com/bsvineeth/activeadmin/blob/29c61de9fb4a5f1ddd743b677f549b725857d0ca/features/action_item.feature#L11